lib/app/features/home/home_widgets/home_options_tap_bar/options_button_design.dart

```diff
@@ -0,0 +1,34 @@
+
+import 'package:flutter/material.dart';
+import 'package:score_live/presentation/constants/app_colors.dart';
+
+class OptionsButtonDesign extends StatelessWidget {
```

Comment:
Why not just **OptionButton**? **OptionsButtonDesign** sounds strange imo

Reply:
applied, naming anything in code is surprisingly hard :sweat_smile: